src/javascript/app_2/Stores/Modules/Portfolio/portfolio_store.js

```diff
@@ -0,0 +1,145 @@
+import {
+    action,
+    computed,
+    observable }                   from 'mobx';
+import { formatPortfolioResponse } from './Helpers/format_response';
+import BaseStore                   from '../../base_store';
+import { WS }                      from '../../../Services';
+import {
+    getDiffDuration,
+    formatDuration }               from '../../../Utils/Date';
+
+export default class PortfolioStore extends BaseStore {
+    @observable data       = [];
+    @observable is_loading = false;
+    @observable error      = '';
+
+    constructor(common) {
```

Comment:
To prevent getting mistaken by local properties and also for consistency, better to use `root_store` everywhere.